cmd/context.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var context = &cobra.Command{
+	Use:     "contexts",
+	Aliases: []string{"ctx", "context"},
+	Short:   "Manage Context configuration",
+	Long:    `Manage Context configuration which allows multiple grafana configs to be used.`,
```

Comment:
nit: Make the quotes on this line consistent with the lines above. I suggest using ".

Reply:
The example code for cobra uses ` ` for the long description.  It's intended to be multiline I imagine, but fixed.

Reply:
Ah. that makes sense. I'm on board with back-ticks then.